Makefile.envs

```diff
@@ -239,8 +240,9 @@ ifeq ($(DISABLE_DYLINK), 1)
 		-s USE_ZLIB \
 		-s USE_BZIP2 \
 		-s FORCE_FILESYSTEM=1 \
-		-s TOTAL_MEMORY=20971520 \
+		-s INITIAL_MEMORY=20971520 \
 		-s ALLOW_MEMORY_GROWTH=1 \
+                -s MAXIMUM_MEMORY=4GB \
```

Comment:
Fix the white space on this line.

Reply:
Fixed it in commit 1e72ba7

Reply:
The diff still shows 8 spaces instead of 2 tabs.

Reply:
Ok, I am confused, can someone help me on this?